.travis.yml

```diff
@@ -16,6 +16,7 @@ install:
 # command to run tests
 script:
  - pep8 mycroft test
+ - ./start.sh unittest --fail-on-error
```

Comment:
does nose2 test not fail on test error? Seems weird we need to run the tests twice here.